_posts/2023-03-09-multilayer-maps.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title:  "Getting started with multilayer maps in OpenSearch"
+authors:
+  - vijay
+  - navneev
+  - junqiu
+  - satnandi
+  - kolchfa
+date:   2023-03-09
+categories:
+  - technical-post
+meta_keywords: 
+meta_description: 
```

Comment:
Please add the following meta: 

**meta_keywords**: multilayer maps, multiple map layers, OpenSearch Dashboards, geospatial data
**meta_description:** Explore the multilayer map functionality in OpenSearch and see how you can visualize geographical data from different data sources at different zoom levels.